cypress/support/fragments/inventory/inventoryInstances.js

```diff
@@ -983,4 +987,23 @@ export default {
       cy.expect([inventoriesList.exists, inventoriesList.has({ rowCount: 1 })]);
     }
   },
+
+  clickSelectAllInstancesCheckbox() {
+    cy.do(selectAllInstancesCheckbox.click());
+    cy.get(Checkbox({ ariaLabel: 'Select instance' })).each(($checkbox) => {
+      cy.expect(Cypress.$($checkbox).checked);
```

Comment:
please use another approach, maybe something like `cy.expect(Checkbox({ name: 'discoverySuppress' }).has({ checked: true }));` or ` cy.get(checkboxSelector).each((checkbox) => {
    cy.expect(checkbox).to.have.attr('checked', 'true');
  });`

Reply:
Changed to 
cy.get(Checkbox({ ariaLabel: 'Select instance' })).each((checkbox) => {
      cy.expect(checkbox.checked);
    });
    since suggested variants didn't work for different reasons. 
    Fixed method for checking if all checkboxes are unchecked the same way